i2c/bmp280_i2c/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(bmp280_i2c
+        bmp280_i2c.c
+        )
+
+# Pull in our (to be renamed) simple get you started dependencies
```

Comment:
oops; did "(to be rename) simple" come from another CMakeLists.txt? Either way, let's fix \

Reply:
https://github.com/raspberrypi/pico-examples/search?q=renamed

Reply:
Got this from the other example's CMakeLists.txt and fixed here.